build-constraints.yaml

```diff
@@ -1983,20 +1895,7 @@ packages:
         - xmlbf
 
     "Tomas Carnecky @wereHamster":
-        - avers
-        - avers-api
-        - avers-server
-        - css-syntax
```

Comment:
@alcinnz, maybe you'd like to take this one over, so that you'd get pings when it breaks? I see you uploaded this package in May 2024.

Reply:
Yup, I'm maintaining css-syntax!

Its stable, so I'm just making sure it doesn't break.

Reply:
@alcinnz If you care about it being in Stackage, you can send a PR to assign it to your own 'section'.

Reply:
Might as well...